pdg/src/snapshots/c2rust_pdg__tests__analysis_test_pdg_snapshot.snap

```diff
@@ -77,12 +77,12 @@ g {
 nodes_that_need_write = []
 
 g {
-	n[0]: copy _ => _46 @ bb31[6]: fn main;
+	n[0]: copy _ => _45 @ bb30[7]: fn main;
```

Comment:
why has the snapshot changed?

Reply:
i see you mentioned this in your description as well -- i'd like to get to the bottom of that

Reply:
my initial guess is that something has changed in the way we compile the program we want to instrument, just need to identify what that is if it's the case

Reply:
It seems like most of the changes are moved one `Local` and one basic block back.  I'm still not sure what I changed to cause these changes.  Perhaps `c2rust_analysis_rt` being a normal dependency and not an `extern`?  That might change the order of crate compilation, as previously we forced `c2rust_analysis_rt` to be compiled first, and now `cargo` just figures out the order it wants.

Reply:
The difference is that now the `--release` arg is now honored, and in the release build, some things are optimized out, resulting in the slightly lower local and basic block numbers.  Now I've switched back to explicitly specifying `PROFILE=debug` so that there's no diff (https://github.com/immunant/c2rust/pull/554/commits/0a46ad82bb4b4fc0a237ad33ddde3adc6f156bc9).  Going forward, I can add a `PROFILE=release` one, too, to track both.